assets/section-image-banner.css

```diff
@@ -318,21 +318,12 @@
 @media screen and (min-width: 750px) {
   .banner--desktop-transparent .banner__box {
     background: transparent;
-    --color-foreground: 255, 255, 255;
-    --color-button: 255, 255, 255;
-    --color-button-text: 0, 0, 0;
     max-width: 89rem;
     border: none;
     border-radius: 0;
     box-shadow: none;
   }
 
-  .banner--desktop-transparent .button--secondary {
-    --color-button: 255, 255, 255;
-    --color-button-text: 255, 255, 255;
-    --alpha-button-background: 0;
```

Comment:
I don't think we should be removing the alpha value here. If the button is using the outline style it should keep a transparent background 🤔 
![](https://screenshot.click/04-11-30dxd-ny04g.png)

Reply:
On mobile it looks good but not on desktop: [recording](https://screenshot.click/04-23-cdj2f-7n92x.gif).